swap/src/protocol/bob/swap.rs

```diff
@@ -377,29 +377,35 @@ async fn run_until_internal(
 pub async fn negotiate(
     state0: crate::protocol::bob::state::State0,
     amounts: SwapAmounts,
-    swarm: &mut EventLoopHandle,
+    event_loop_handle: &mut EventLoopHandle,
```

Comment:
Minor nit: I would order the commits as such that the feature is in the end.

Reply:
> Minor nit: I would order the commits as such that the feature is in the end.

I tried but it involved merging work that I was not willing to do.